doc/conf.py

```diff
@@ -340,5 +340,11 @@ def setup(app):
     'within_subsection_order': NumberOfCodeLinesSortKey,
     'find_mayavi_figures': find_mayavi_figures,
     'expected_failing_examples': ['../examples/no_output/plot_raise.py',
-                                  '../examples/no_output/plot_syntaxerror.py']
+                                  '../examples/no_output/plot_syntaxerror.py'],
+    'binder': {'org': 'sphinx-gallery',
+               'repo': 'sphinx-gallery.github.io',
+               'url': 'https://mybinder.org',
+               'branch': 'gh-pages',
```

Comment:
This has to be master for the website. Was this a problem of git?

Reply:
already fixed in #326 , was just my mistake

Reply:
(needs a merge tho)